targets/common

```diff
@@ -32,6 +32,19 @@ if [ "$TARGETS" = 'help' ]; then
     if [ -n "$REQUIRES" ]; then
         echo "	Requires: $REQUIRES"
     fi
+elif [ "$TARGETNOINSTALL" = "p" ]; then
+    # Avoid adding things already provided
+    # This will only check the provides from /etc/crouton/targets
+    # and targets specified with -t.
+    for t in $PROVIDES; do
+        # Don't add duplicate PROVIDES. If there are duplicates
+        # we will respect the order of the targets specified.
+        # This means on an update targets specified with -t
+        # will have higher priority then already installed targets.
+        if ! grep -q "^$t=.*\$" "${PROVIDESFILE:-/dev/null}"; then
```

Comment:
There's really no harm in adding duplicate provides if your awk statement down below has `print $2; exit`.  First one will naturally take precedence.